src/Audit/ModuleDisabled.php

```diff
@@ -39,7 +39,7 @@ public function audit(Sandbox $sandbox)
 
     $status = strtolower($info[$module]['status']);
 
-    return ($status == 'not installed');
+    return ($status != 'enabled');
```

Comment:
We need to maintain backwards compatibility so we to ensure this works for < 8.5.6 too.

Reply:
I'm reasonably certain that rewriting the conditional this way will work in older versions as well. I don't have a D7 site so I'm spinning one up on free-tier to test with.

Reply:
Confirmed. D7 reports it as `enabled` so if we check for `$status != 'enabled'` it will work for both D8 and D7.

```
    "field_sql_storage": {
        "package": "Core",
        "name": "Field SQL storage (field_sql_storage)",
        "type": "Module",
        "status": "Enabled",
        "version": "7.59"
    },
```

Reply:
D7 doesn't matter so much since this is a plugin explicitly for D8. I was more concerned about previous versions. Happy to commit. Thanks!

Reply:
Quick note: In 2.2 and 2.3 I'm seeing all the checks come back showing the mod is installed, even when it isn't. Shouldn't I be seeing the fixed version now?